src/dex/nerve/config.ts

```diff
@@ -0,0 +1,467 @@
+              decimals: 8,
+            },
+            {
+              address: '0xfE18be6b3Bd88A2D2A7f928d00292E7a9963CfC6', // 2 - sBTC
+              decimals: 18,
+            },
+          ],
+          isMetapool: false,
+          isUSDPool: false,
+          lpToken: {
+            address: '0xF32E91464ca18fc156aB97a697D6f8ae66Cd21a3',
+            decimals: 18,
+          },
+        },
+        alETH: {
+          name: 'alETH',
+          address: '0xa6018520EAACC06C30fF2e1B3ee2c7c22e64196a',
+          coins: [
+            {
+              address: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2', // 0 - WETH
+              decimals: 18,
+            },
+            {
+              address: '0x0100546F2cD4C9D97f798fFC9755E47865FF7Ee6', // 1 - alETH
+              decimals: 18,
+            },
+            {
+              address: '0x5e74C9036fb86BD7eCdcb084a0673EFc32eA31cb', // 2 - sETH
+              decimals: 18,
+            },
+          ],
+          isMetapool: false,
+          isUSDPool: false,
+          lpToken: {
+            address: '0xc9da65931ABf0Ed1b74Ce5ad8c041C4220940368',
+            decimals: 18,
+          },
+        },
+        StablecoinV2: {
+          name: 'StablecoinV2',
+          address: '0xaCb83E0633d6605c5001e2Ab59EF3C745547C8C7',
+          coins: [
+            {
+              address: '0x6B175474E89094C44Da98b954EedeAC495271d0F', // 0 - DAI
+              decimals: 18,
+            },
+            {
+              address: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48', // 1 - USDC
+              decimals: 6,
+            },
+            {
+              address: '0xdAC17F958D2ee523a2206206994597C13D831ec7', // 2 - USDT
+              decimals: 6,
+            },
+          ],
+          isMetapool: false,
+          isUSDPool: true,
+          lpToken: {
+            address: '0x5f86558387293b6009d7896A61fcc86C17808D62',
+            decimals: 18,
+          },
+        },
+      },
+    },
+    // [Network.FANTOM]: {
+    //   poolConfigs: {
+    //     FtmUSD: {
+    //       name: 'FtmUSD',
+    //       address: '0xBea9F78090bDB9e662d8CB301A00ad09A5b756e9',
+    //       coins: [
+    //         {
+    //           address: '0xdc301622e621166BD8E82f2cA0A26c13Ad0BE355', // 0 - FRAX
+    //           decimals: 18,
+    //         },
+    //         {
+    //           address: '0x04068DA6C83AFCFA0e13ba15A6696662335D5B75', // 1 - USDC
+    //           decimals: 6,
+    //         },
+    //       ],
+    //       isMetapool: false,
+    //       isUSDPool: true,
+    //       lpToken: {
+    //         address: '0xc969dd0a7ab0f8a0c5a69c0839db39b6c928bc08',
+    //         decimals: 18,
+    //       },
+    //     },
+    //   },
+    // },
+  },
+};
+
+export const dexKeyToABIMap: Record<string, AbiItem[]> = {
+  Nerve: nervePoolABI,
+  Axial: axialPoolABI,
+  IronV2: ironV2PoolABI,
+  // The same as axial
+  Saddle: axialPoolABI,
+};
```

Comment:
You can also put this in the DexConfigMap. Also did you check if the ABI is different? If they are forks the ABI might be exactly same

Reply:
Yes, put the `abi` property to config. Seems good